pkg/util/log/clog.go

```diff
@@ -930,6 +931,16 @@ func (sb *syncBuffer) rotateFile(now time.Time) error {
 		return err
 	}
 
+	// Redirect stderr to the current INFO log file in order to capture panic
```

Comment:
I think that with our current use of log levels, we're not getting much value out of the multi-file logging. Whenever we investigate an issue, we look at the INFO log (or often, we restart with extra `--vmodule` flags); we never (or at least I never) start with the ERROR or WARNING files for less noise/more history. (and many users prefer `--logtostderr` anyway). So I'd vote to collapse everything into a single log file (at least until some hypothetical future release when we are better able to classify the interesting/low-volume messages as errors or warnings). That can be a separate change; for now just dump it into INFO.